package.json

```diff
@@ -9,7 +9,8 @@
     "lint": "eslint src --fix",
     "test": "jest",
     "test:watch": "jest --watch",
-    "test:coverage": "jest --collectCoverage"
+    "test:coverage": "jest --collectCoverage",
+    "scss": "node-sass scss/style.scss assets/css/style.css --watch"
```

Comment:
You should configure webpack to handle this for you instead of running a script manually